cfgov/unprocessed/js/modules/util/data-hook.js

```diff
@@ -60,8 +60,22 @@ function contains( element, value ) {
   return values.indexOf( value ) > -1 ? true : false;
 }
 
+/**
+ * @param {string} behavior
+ *  Behavior type used to find the element within the dom.
+ * @returns {HTMLNode} if it exists in the dom, null otherwise.
+ */
+function get( behavior ) {
```

Comment:
Can we name this `getElementFor` or `getElemFor` or something? `get` is generic and may imply this is getting the data-hook (`data-js-hook`).